policies/io/konveyor/forklift/ovirt/cpu_policy.rego

```diff
@@ -0,0 +1,17 @@
+package io.konveyor.forklift.ovirt
+
+default not_supported_cpu_policy = false
+
+# manual pinning validation is under cpu_tune policy.
```

Comment:
how about "no need to check for 'manual' pinning policy because 'cpuAffinity' is validated by the 'cpu_tune' policy"

Reply:
sounds good